index.php

```diff
@@ -13,7 +13,7 @@
 
 class Webgrind_MasterConfig
 {
-    static $webgrindVersion = '1.7';
+    static $webgrindVersion = '1.7.1';
```

Comment:
Could you remove this change?  I will bump the version on release.

Reply:
Done